commands/utils/params.go

```diff
@@ -347,3 +369,27 @@ func generateConfigAggregatorFromEnv(gitParams *Git, server *coreconfig.ServerDe
 	repo.Projects = append(repo.Projects, project)
 	return &FrogbotConfigAggregator{repo}, nil
 }
+
+func DownloadConfigFromTarget(client vcsclient.VcsClient) ([]byte, error) {
```

Comment:
Looks like this function can be made private.
Let's also add a description for this function.

Reply:
done